art/decor_dic.py

```diff
@@ -57,7 +57,7 @@
 fancy27 = [").•ˆ•+*¨", "¨*+•ˆ•.("]
 fancy28 = ["•·.·¯`·.·•", "•·.·¯`·.·•"]
 fancy29 = ["•·.·´¯`·.·•", "•·.·´¯`·.·•"]
-fancy3 = ["·٠•●♥ Ƹ̵̡Ӝ̵̨̄Ʒ ♥●•٠·˙" ,"˙·٠•●♥ Ƹ̵̡Ӝ̵̨̄Ʒ ♥●•٠·˙"]
+fancy3 = ["·٠•●♥ Ƹ̵̡Ӝ̵̨̄Ʒ ♥●•٠·˙", "˙·٠•●♥ Ƹ̵̡Ӝ̵̨̄Ʒ ♥●•٠·˙"]
```

Comment:
I think `,` should come after `"`

Reply:
It's a little weird!!  
Because I see this picture : 
![image](https://user-images.githubusercontent.com/7515099/94996146-b3ddb800-05af-11eb-8f01-27b46cc07b34.png)

I think it something related to `RTL` , `LTR` preview!!